CMakeLists.txt

```diff
@@ -1,21 +1,24 @@
-#=============================================================================
-#  MuseScore
-#  Linux Music Score Editor
-#
-#  Copyright (C) 2002-2016 by Werner Schweer and others
-#
-#  This program is free software; you can redistribute it and/or modify
-#  it under the terms of the GNU General Public License version 2.
-#
-#  This program is distributed in the hope that it will be useful,
-#  but WITHOUT ANY WARRANTY; without even the implied warranty of
-#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#  GNU General Public License for more details.
-#
-#  You should have received a copy of the GNU General Public License
-#  along with this program; if not, write to the Free Software
-#  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
-#=============================================================================
+#[[
+// SPDX-License-Identifier: GPL-3.0-only
```

Comment:
Why not just replace `//` with `#` on all lines for CMakeLists.txt, *.cmake, shell scripts, and Python files?

Reply:
This is basically a result of automated work from external utility, so I don't really care about the specific formatting cases

Reply:
I can only think of two cases though:

1. Files that use `//` as comments (`.c`, `.h`, `.cpp`, `.hpp`)
2. Files that use `#` as comments (`.sh`, `.bash`, `.py`, `.cmake`, `CMakeLists.txt`)

So it might worth writing a simple shell script to add the license header based on extention. I could do this if you don't have time.

But if you want to use the existing formatter then all you need to do is remove the `//` from each line and pass in the license text on its own:

```
SPDX-License-Identifier: GPL-3.0-only
MuseScore-CLA-applies
=============================================================================
MuseScore
Music Composition & Notation

Copyright (C) 2021 MuseScore BVBA and others

This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License version 3 as
published by the Free Software Foundation.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program.  If not, see <https://www.gnu.org/licenses/>.
=============================================================================
```

The problem currently is that each line starts with `//` so the formatter things that is supposed to be *part of* the comment rather than the escape sequence to introduce the comment. This is why the `.cpp` files are getting `/**` added before the comment.